docs/en/docs/sql-manual/sql-functions/window-functions/WINDOW-FUNCTION-WINDOW-FUNNEL.md

```diff
@@ -0,0 +1,62 @@
+---
+{
+    "title": "WINDOW-FUNCTION-WINDOW_FUNNEL",
+    "language": "en"
+}
+---
+
+<!--  Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.  See the NOTICE file distributed with this work for additional information regarding copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License. -->
+
+## WINDOW FUNCTION WINDOW_FUNNEL
+### description
+
+Searches the longest event chain happened in order (event1, event2, ... , eventN) along the timestamp_column with length of window.
+
+- window is the length of time window in seconds.
+- mode is reserved for future, not used for now.
+- timestamp_column specifies column of DATE or DATETIME type.
```

Comment:
I think we should change `timestamp_column` to only DATETIME type because the granularity of time window is second now